application/src/test/resources/org/opentripplanner/apis/gtfs/expectations/realtime-trip.json

```diff
@@ -3,13 +3,9 @@
     "addedTrip": {
       "gtfsId": "F:ADDED_TRIP",
       "timetabledOrigin": null,
-      "datedOrigin": {
-        "realtimeDeparture": 0
-      },
+      "datedOrigin": null,
       "timetabledDestination": null,
-      "datedDestination": {
-        "realtimeArrival": 900
-      },
+      "datedDestination": null,
```

Comment:
These changes are slightly unexpected. I would have thought this pr only increases the number of cases where data is found?

Reply:
This change is certainly unwarranted. I am testing the data should be returned on `ADDED_TRIP` on the day it does run.

Reply:
I have read further, that the pre-6245 logic for departureTime and arrivalTime was wrong.

It first fetched the trip pattern (line 134, without referring to the date), and added the date no matter which date it was. Therefore it didn't process added or replacement trips correctly.